### ISSUE TITLE
odd more information guide linking. merge into atomicapp/docs?

### ISSUE THREAD

**Issue by charliedrage** [projectatomic/atomicapp#251]
If you click on the *Atomic App Getting started Guide* within the `README.md` it takes you to the page on *projectatomic.io* which is almost identical to the `README.md` of atomicapp.

Once there you have to click on *Nulecule getting started guide* to finally get to the page for more information.

Wanted to open this issue for discussion. Do we simply update the README.md with the corresponding link. Or shall we add a getting started guide to atomicapp/docs?

**Comment by goern** [comment]
From my point of view the flow should be more like [Nulecule getting started](https://github.com/projectatomic/nulecule/blob/master/docs/getting-started.md) and later on get practical with [Atomic App getting started](http://www.projectatomic.io/docs/atomicapp/) and we should link again aarons blog post. 

All docs should be hosted on projectatomic.io and READMEs on repos should point that containing some abstracts.

We should also think about DevAssistant and ADB/BCL/BCJ and the [BBC](http://britishbeer.com/westford/)

@bexelbie @aweiteka @vpavlin

**Comment by bexelbie** [comment]
I agree that the docs need a presence on projectatomic.io, however I don't think the sources should live there.  I am concerned that if we put the sources in the website we 1) limit the number of commiters artificially and 2) make it even easier to do commits without docs.

Therefore I would propose that we leave the docs in the source repos and point to them from projectatomic.io.  For now they are not so large as to demand a full "docs site" treatment.  When they are we can revisit this (and probably create docs.projectatomic.io),

This still leaves #223 requiring a solution.  The relationship between all the parts is getting very confusing for new people.

**Comment by goern** [comment]
maybe we can consolidate all the developer facing howto docs in `docs/` (of atomicapp and nulecule) and shorten the readme, and push from `docs/` to projectatomic.io/....

**Comment by bexelbie** [comment]
@goern I was thinking along those lines as well.  @jzb and crew will need to decide how rebuilds of projectatomic.io should be managed.  I don't think they have CI on that yet .. but if we did sub-repositories or some other wizadry we could probably make it work without having to increase the number of website committers.  Or we could add some committers to the site who stayed in the docs field only.

**Comment by cdrage** [comment]
Fixed after the docs refactor :)

Although incoming PR's to update atomic-site @jberkus with an update on how to use atomicapp / nulecule.

**Comment by jberkus** [comment]
FYI: we are moving to a new model where docs will live at docs.projectatomic.io.  This will allow you to push directly from your own docs repo to there without needing to add docs to atomic-site.

This should be ready in the next 2 weeks, and then we'll talk about getting the atomic.app and Nulecule docs in.
